engine/src/main/java/org/hibernate/search/query/dsl/sort/MultiValuedMode.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ *
+ */
+package org.hibernate.search.query.dsl.sort;
+
+/**
+ * Mode when comparing multiple values.
+ *
+ * @author Emmanuel Bernard emmanuel@hibernate.org
+ */
+public enum MultiValuedMode {
```

Comment:
Nice, hadn't thought about multi-values. I'm afraid you'll have to pick a default, so that it can work even if the user didn't  invoke the method `org.hibernate.search.query.dsl.sort.SortFieldContext.treatMultiValuedAs(MultiValuedMode)` ?